### PR TITLE
Created API implemented it

### LINKED ISSUE #36: API for group-subgroup and implement it in UI
API to be made for getting group and subgroup list which is to be displayed on the group/subgroup page on the admin side.
**TASK:**
To create query, function, and route on the server side and test it using postman
**Outcome:** the result should be a JSON file containing all the group details and another JSON file containing all subgroup details

### PR DESCRIPTION
**Pull request by Kahitoz**
Please check and merge

### CODE REVIEW

**Comment by palashmunshi** [comment]
Please let me know when your work is ready for review
Thanks.

**Comment by Kahitoz** [comment]
My work is complete, I was not able to make the popup, kindly figure it out rest I have make some interesting changes in ui be sure to explore it thanks, 
Regards Kshitiz

File: client-side/src/Components/adminComponent/Settings/group.js [hunk 1]
```diff
@@ -89,6 +98,7 @@ export const Group = () => {
           </div>
          
         </div>
+        <p>Note- The table is fucked</p>
```
Comment (palashmunshi) [inline client-side/src/Components/adminComponent/Settings/group.js]: change this line please.

File: client-side/src/Components/adminComponent/Settings/group.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ import "./group.scss";
 export const Group = () => {
   const [groups, setGroups] = useState([]);
   const [subgroups, setSubgroups] = useState([]);
+  const counter = () =>{
+    let counter1 = 0;
```
Comment (palashmunshi) [inline client-side/src/Components/adminComponent/Settings/group.js]: change the variable name from counter1 to something else

File: client-side/src/Components/adminComponent/Settings/group.js [hunk 3]
```diff
@@ -19,6 +24,10 @@ export const Group = () => {
       .then((response) => response.json())
       .then((data) => setSubgroups(data))
       .catch((error) => console.log(error));
+
+
```
Comment (palashmunshi) [inline client-side/src/Components/adminComponent/Settings/group.js]: remove extra space

File: client-side/src/Components/adminComponent/Settings/group.js [hunk 4]
```diff
@@ -1,36 +1,47 @@
...
 export const Group = () => {
   const [groups, setGroups] = useState([]);
   const [subgroups, setSubgroups] = useState([]);
-  const counter = () =>{
-    let counter1 = 0;
-    counter1++;
-    return counter1;
-  }
+  const [showMore, setShowMore] = useState(false);
+  const [showMore1, setShowMore1] = useState(false);
```
Comment (palashmunshi) [inline client-side/src/Components/adminComponent/Settings/group.js]: change variable names to showMoreGroup and showMoreSubgroup

**Review by palashmunshi — CHANGES_REQUESTED** [review]
Please read all 4 comments.

File: client-side/src/Components/adminComponent/Settings/group.js (hunk 4 above)
Comment (Kahitoz) [reply client-side/src/Components/adminComponent/Settings/group.js]: Hi I have done it as per your request, please accept the changes

Comment (Kahitoz) [reply client-side/src/Components/adminComponent/Settings/group.js]: Corrected the issue, Thanks

**Review by palashmunshi — APPROVED** [review]
nice

**Pull request merged**